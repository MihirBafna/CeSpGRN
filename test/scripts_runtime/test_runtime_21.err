../../src/g_admm.py:45: UserWarning: torch.eig is deprecated in favor of torch.linalg.eig and will be removed in a future PyTorch release.
torch.linalg.eig returns complex tensors of dtype cfloat or cdouble rather than real tensors mimicking complex tensors.
L, _ = torch.eig(A)
should be replaced with
L_complex = torch.linalg.eigvals(A)
and
L, V = torch.eig(A, eigenvectors=True)
should be replaced with
L_complex, V_complex = torch.linalg.eig(A) (Triggered internally at  /opt/conda/conda-bld/pytorch_1631630836880/work/aten/src/ATen/native/BatchLinearAlgebra.cpp:2897.)
  E, V = torch.eig(A, eigenvectors = False)
../../src/g_admm.py:541: UserWarning: torch.cholesky is deprecated in favor of torch.linalg.cholesky and will be removed in a future PyTorch release.
L = torch.cholesky(A)
should be replaced with
L = torch.linalg.cholesky(A)
and
U = torch.cholesky(A, upper=True)
should be replaced with
U = torch.linalg.cholesky(A.transpose(-2, -1).conj()).transpose(-2, -1).conj() (Triggered internally at  /opt/conda/conda-bld/pytorch_1631630836880/work/aten/src/ATen/native/BatchLinearAlgebra.cpp:1284.)
  ll = torch.cholesky(Z)
