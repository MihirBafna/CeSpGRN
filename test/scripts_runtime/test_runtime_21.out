ngenes: 50, seed: 1
number of nearest neighbor: 6
number of nearest neighbor: 7
number of nearest neighbor: 8
number of nearest neighbor: 9
final number of nearest neighbor (make connected): 9
number of neighbor being considered: 30
time calculating the kernel function: 256.30 sec
time cost (estimating empirical covariance matrix): 256.30 sec
start running batch 0
n_iter: 500, duality gap: 5.2816e-08, primal residual: 6.6777e-07, dual residual: 9.810160e-07
Batch loss: loss1: 4130.15674, loss2: 13078.12305, loss3: 0.00000
Finished running batch 0, running time: 42.35 sec
start running batch 1
n_iter: 500, duality gap: 2.9935e-08, primal residual: 3.1737e-07, dual residual: 7.321241e-07
Batch loss: loss1: 4328.14551, loss2: 12124.22266, loss3: 0.00000
Finished running batch 1, running time: 42.30 sec
start running batch 2
n_iter: 500, duality gap: 1.8359e-08, primal residual: 2.4658e-07, dual residual: 6.984801e-07
Batch loss: loss1: 4402.91260, loss2: 11796.22852, loss3: 0.00000
Finished running batch 2, running time: 41.92 sec
start running batch 3
n_iter: 500, duality gap: 2.5237e-08, primal residual: 2.3274e-07, dual residual: 8.489418e-07
Batch loss: loss1: 4390.58789, loss2: 12052.33691, loss3: 0.00000
Finished running batch 3, running time: 39.92 sec
start running batch 4
n_iter: 500, duality gap: 1.8294e-08, primal residual: 2.1454e-07, dual residual: 6.437620e-07
Batch loss: loss1: 4314.02588, loss2: 12746.94922, loss3: 0.00000
Finished running batch 4, running time: 40.71 sec
start running batch 5
n_iter: 500, duality gap: 2.5310e-08, primal residual: 2.3110e-07, dual residual: 6.831582e-07
Batch loss: loss1: 4183.19336, loss2: 13678.79883, loss3: 0.00000
Finished running batch 5, running time: 42.32 sec
start running batch 6
n_iter: 500, duality gap: 2.3696e-08, primal residual: 2.3693e-07, dual residual: 7.674973e-07
Batch loss: loss1: 4398.64551, loss2: 11829.91016, loss3: 0.00000
Finished running batch 6, running time: 41.22 sec
start running batch 7
n_iter: 500, duality gap: 2.1025e-08, primal residual: 2.4134e-07, dual residual: 6.498387e-07
Batch loss: loss1: 4375.89648, loss2: 12124.02344, loss3: 0.00000
Finished running batch 7, running time: 40.88 sec
start running batch 8
n_iter: 500, duality gap: 2.3011e-08, primal residual: 2.3018e-07, dual residual: 7.052098e-07
Batch loss: loss1: 4289.64258, loss2: 12833.02930, loss3: 0.00000
Finished running batch 8, running time: 42.63 sec
start running batch 9
n_iter: 500, duality gap: 2.1562e-08, primal residual: 2.1598e-07, dual residual: 6.259915e-07
Batch loss: loss1: 4145.13916, loss2: 13874.55176, loss3: 0.00000
Finished running batch 9, running time: 42.02 sec
time cost (estimating theta): 417.71 sec
